package Protos;

message AdaptIntegrateAndFireC {
    required double tau_m = 1;
    required double rest_pot = 2;
    required double tau_ref = 3;
    required double noise = 4;
    required double tau_adapt = 5;
    required double kd = 6;
    required double vK = 7;
    required double adapt_amp = 8;
    required double gKCa = 9;
}

message AdaptIntegrateAndFireState {
    required double p = 1;
    required double u = 2;
    required double ref_time = 3;
    required double Ca = 4;
}

